backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/Linker.kt

```diff
@@ -18,6 +19,13 @@ internal fun determineLinkerOutput(context: Context): LinkerOutputKind =
             else -> TODO("${context.config.produce} should not reach native linker stage")
         }
 
+// TODO: Move to a more appropriate place.
+internal val KonanTarget.isAppleTarget: Boolean
+        get() = family == Family.IOS || family == Family.OSX
+
+internal val KonanTarget.isTvOs: Boolean
```

Comment:
`isTvOS`?

Reply:
Renamed to `isTvOsBased`. Will be removed with introduction of `Family.TVOS`.

Reply:
No, `isTvOS` is good. I mean that `s` should probably be capital.

Reply:
Ok. Anyway, `isTvOsBased` feels a bit better since target is not an OS :)